app/reducers/app_reducer.js

```diff
@@ -0,0 +1,10 @@
+import {combineReducers} from 'redux';
+import {channels} from './channels.js';
+import {messages} from './messages.js';
+
+export function appReducer(state, action) {
+  switch (action.type) {
+  default:
+    return combineReducers({channels, messages});
+  }
+}
```

Comment:
Don't clearly understand why you need check action type here. Here you just need to combine all reducers in one.